go-controller/pkg/clustermanager/userdefinednetwork/controller.go

```diff
@@ -674,6 +675,13 @@ func (c *Controller) updateClusterUDNStatus(cudn *userdefinednetworkv1.ClusterUs
 	return nil
 }
 
+func (c *Controller) EnsureDefaultNetworkNAD() error {
+	if _, err := defaultnetworkcustomization.EnsureDefaultNetworkNAD(c.nadLister, c.nadClient); err != nil {
+		return fmt.Errorf("unable to ensure default network nad exists: %w", err)
+	}
```

Comment:
Let's use "failed" prefix like 90% of errors do that and is easier to grep
```
failed ensuring...
``

Reply:
Done.